parquet-column/src/main/java/org/apache/parquet/column/statistics/SizeStatistics.java

```diff
@@ -67,8 +67,16 @@ public static class Builder {
     private Builder(PrimitiveType type, int maxRepetitionLevel, int maxDefinitionLevel) {
       this.type = type;
       this.unencodedByteArrayDataBytes = 0L;
-      repetitionLevelHistogram = new long[maxRepetitionLevel + 1];
-      definitionLevelHistogram = new long[maxDefinitionLevel + 1];
+      if (maxRepetitionLevel > 0) {
+        repetitionLevelHistogram = new long[maxRepetitionLevel + 1];
+      } else {
+        repetitionLevelHistogram = new long[0]; // omitted
```

Comment:
not sure if there is a style for preferring null vs 0 length array (or does this flow through more easily for consuption)?

Reply:
0 length array makes it easier for other methods (e.g. `copy()`) to avoid `NullPointerException`.